syntax = "proto3";

package recommendations;

import "protos/participants.proto";

message GetRecommendationsRequest {
  string user_id = 1;
  repeated Participants participants = 2;
}

message GetRecommendationsResponse {
  repeated string recommendations = 1;
}

service Recommendations {
  rpc GetRecommendations (GetRecommendationsRequest) returns (GetRecommendationsResponse) {}
}